modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/DefaultExecutionMediator.java

```diff
@@ -154,9 +168,9 @@ private InterceptorsExecutionResult before(ExecutionContext executionContext, Li
   private void onSuccess(ExecutionContext executionContext, Object result, List<Interceptor> interceptors) {
     intercept(interceptors,
               interceptor -> interceptor.onSuccess(executionContext, result), interceptor -> format(
```

Comment:
AFAIK is better than concatenating. Also consider that the format will only be executed upon errors which will be rare at this point. I don't think it makes an actual differnece